### DISCUSSION TITLE
為什麼人類都能發明火箭上太空，卻還在用幾千年發明的雨傘避雨

### DISCUSSION THREAD

**Discussion by zoo100130** [1101-GS4719-C/Python-Community-GS4719-C#2129] [General]
發現有些功能跟需求，儘管科技進步了幾千年卻還是保有原來的樣子，有些工具甚至在那些科學還沒被發揚的年代就沿用至今，不禁去想一下，在下一個千年，這個時代的發明有哪些還能保有原樣留存下去

**Comment by Jing-Asteroid** [comment]
推薦你去上本校黃金俊老師的創造力課程，課堂上會討論到如何創新的東西，改善物品的使用。
雨傘的這個議題我記得他也有在課堂上提過，也有讓我們去討論可以如何創新，是個很ㄈ...很涼、很甜又可以促進思考的課喔~~
